polyply/tests/test_simple_seq_parsers.py

```diff
@@ -20,37 +20,54 @@
 from polyply import TEST_DATA
 from polyply.src.meta_molecule import MetaMolecule
 from .example_fixtures import example_meta_molecule
-from polyply.src.simple_seq_parsers import (_identify_nucleotypes,
+from polyply.src.simple_seq_parsers import (_identify_residues,
                                             _monomers_to_linear_nx_graph,
                                             _parse_plain,
                                             FileFormatError)
 
-@pytest.mark.parametrize('comments, DNA, RNA', (
+@pytest.mark.parametrize('comments, DNA, RNA, PROTEIN', (
     # single DNA comment
       (["DNA lorem ipsum"],
        True,
+       False,
        False
       ),
     # single RNA comment
       (["RNA lorem ipsum"],
        False,
-       True
+       True,
+       False
       ),
     # single DNA comment multiple lines
       (["lorem ipsum", "random line DNA", "DNA another line"],
        True,
+       False,
        False
       ),
     # single RNA comment multiple lines
       (["lorem ipsum", "random line RNA", "RNA another line"],
+       False,
+       True,
+       False
+      ),
+    # single AA comment
+      (['lorem ipsum PROTEIN'],
+       False,
        False,
        True
       ),
-     ))
-def test_identify_nucleotypes(comments, DNA, RNA):
-    out_DNA, out_RNA = _identify_nucleotypes(comments)
+    # singe AA comment multiple lines
+      (["lorem ipsum", "random line PROTEIN", "PROTEIN another line"],
+       False,
+       False,
+       True
+      ),
+      ))
+def test_identify_nucleotypes(comments, DNA, RNA, AA):
```

Comment:
the name of the input argument here needs to be the same as in line 28. That's why the test is failing. So either change AA -> PROTEIN or the other way around

Reply:
ah thanks for spotting, I knew I'd get into an issue with this.

Reply:
hi Chris looks good overall. now the only complaint is that there is an untested line of code. let me investigate